### ISSUE TITLE
High CPU usage due to supervisord

### ISSUE THREAD

**Issue by Dvlnst** [ONLYOFFICE/DocumentServer#772]
**Do you want to request a *feature* or report a *bug*?**
A suspected bug.

**What is the current behavior?**
High CPU usage by supervisor.service.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Before starting supervisord service, cpu usage for each logical cpu are between 0-2%, when I start supervisor, it's over 20% handling no clients. I read there are high cpu usage problems with supervisor for a long time related with log files ([see](https://github.com/Supervisor/supervisor/issues/807)). Could you check it?

**What is the expected behavior?**
Less CPU usage?

**Did this work in previous versions of DocumentServer?**
I don't know.

**DocumentServer version:** 
5.4.2-46 (installed from debian repository).

**Operating System:** 
Debian 10.

I also tried this #692 but it didn't work.

Kind regards.

**Comment by ShockwaveNN** [comment]
Are you sure it supervisor process is used all that cpu, not some supervisor child?

**Comment by Dvlnst** [comment]
I found the problem. Spellchecker was restarting all the time that's why I wasn't seeing a high CPU usage by a supervisor child. I realized because I saw how PID changed constantly.

I tried to rebuild spellchecker with nodejs 13.7 but same problem as #652. Finally, I downgrade to nodejs 10.19 and it worked as explained in [#312](https://github.com/ONLYOFFICE/DocumentServer/issues/312#issuecomment-392479211).

Kind regards.
